src/demo/java/org/junitpioneer/jupiter/ExpectedToFailExtensionDemo.java

```diff
@@ -18,7 +18,7 @@ public class ExpectedToFailExtensionDemo {
 
 	// tag::expected_to_fail[]
 	@Test
-	@ExpectedToFail
+	@ExpectedToFail(onExceptions = Exception.class)
```

Comment:
Sorry, why did you change this?

Reply:
Hmm... That looks like a mistake. I think I probably did that when I first started looking into this and mistakenly edited this file. Sorry about that!